rules/Symfony30/Rector/ClassMethod/RemoveDefaultGetBlockPrefixRector.php

```diff
@@ -118,6 +114,14 @@ public function refactor(Node $node)
         return null;
     }
 
+    private function camelToSnake(string $content): string
+    {
+        return strtolower(
```

Comment:
@TomasVotruba class name allow contains unicode, so it needs to use `mb_strtolowe()`, see https://3v4l.org/jQJGM

Reply:
https://github.com/rectorphp/rector-symfony/pull/512